rpm/busybox.spec

```diff
@@ -453,7 +453,9 @@ rm -f %{buildroot}/bin/pidof
 %{_bindir}/test
 %{_bindir}/touch
 %{_bindir}/tr
+%{_bindir}/tree
```

Comment:
I quite like having `tree` available, but it's not in `gnu-coreutils`. I wonder how we should deal with that. I guess it's fine if we don't provide it otherwise at all in Sailfish, but it would we the first command that's in `busybox-symlinks-coreutils` that's not in `gnu-coreutils` (I didn't count `[[` though, which is usually provided internally by shell/bash). However if there is some package, then that will conflict and explicit `Conflicts` is needed.

Reply:
It might make sense to enable less at the same time.

Reply:
I removed the new programs from build